cp_wheels.sh

```diff
@@ -66,6 +66,9 @@ function cp_wheel {
 			exit
 		fi
 	fi
+	if [[ "$COMPAT" == "generic" && "$ARCHITECTURE" != "generic" ]]; then
```

Comment:
Can you add a comment in the code explaining what and why ?

Reply:
How is this?

Reply:
LGTM